### ISSUE TITLE
Complex SFM subentry markers

### ISSUE THREAD

**Issue by WesPeacock** [sil-dictionary-lexical-services/SubentryPromotion#3]
The complex subentry types use a MDF marker improperly.
The MDF standard doesn't specify complex subentry types.
The SubentryPromotion repo extends it using \l* markers.
This clashes with other MDF markers, particularly \lc as citation form vs compound subentry

**Comment by WesPeacock** [comment]
rather than \lX (ld,lc,li) we should use \sed \sec \sei for derivative, compound, idiom, etc.

**Comment by WesPeacock** [comment]
Robert H mentioned that \sc is scientific name and is mentioned in the EndMarkers of Sample-lc.ini.
However, it can occur at both the entry and subentry level e.g:

> \lx breath
> 
> \sc respiration
> ...
> \sec Baby's breath
> \sc Gypsophila

This would mean that it shouldn't be in the EndMarkers

**Comment by bbryson** [comment]
We need to decide if we are using PLB markers in the “complex” example.

 - On one hand, I think there is value to it, because that is the most likely scenario in which we will find more than one complex form type.  (Second mostly likely:data that doesn’t use MDF.  Least likely, data that does use MDF.)

 - On the other hand, do we want people to have to learn how to extrapolate?

If we are using PLB markers, then we should not worry about clashing with MDF.  Maybe all we need to do is point out cases where there is a clash (e.g., \ms instead of \sn,  \sn instead of \sc, \sc instead of ??, \lc means “subentry compound” not “citation form”).  But if the dataset is designed for PLB data, we should not assume there would be a mixture of both PLB and MDF markers.

**Comment by WesPeacock** [comment]
No, I think I would recommend against it. The Complex form mapping file is forked from the standard MDF SFM import map.

I would base a PLB solution on a PLB FLEx SFM import map with its own custom se2lx INI file.

I notice that the FLEx installation doesn't include a PLB import map file. We probably need to develop that.
